### ISSUE TITLE
capability of POSTing more than one context event per POST in the RESTAPI

### ISSUE THREAD

**Issue by amedranogil** [universAAL/remote#1]
There can be problems in 3G networks were many connections (in case of sending many context events) can be blocked. 
As a solution the client could send more than one context event in a single POST.

This could be implemented using the SerializerEX, or sending a list of CEs instead of a single CE then the RESTAPI would interpret and unwrap if necessary.

**Comment by Alfiva** [comment]
Should be easy to achieve bundling several CE in a single POST, but have to look into it.

**Comment by cstockloew** [comment]
From a serialization point of view, it's easy: create a dummy root resource with a dummy property containing a list with the events:

```
    private static final String TYPE_AGG_EVENT = "..";
    private static final String PROP_AGG_EVENTS = "..";

```
sender:
```
    Resource r = new Resource();
    r.addType(TYPE_AGG_EVENT, true);
    ArrayList l = new ArrayList();
    l.add(...);    // add all events
    r.setProperty(PROP_AGG_EVENTS, l);

```
receiver:
```
   if (TYPE_AGG_EVENT.equals(r.getType())) {
      // handle agg event
   } else {
      // handle single event
   }

```

**Comment by Alfiva** [comment]
Implemented in c261a3facbf950fec4623659be2a4d5ab505e5d2 
The approach used is bundling them in the HTTP body structure rather than the RDF graph.
If you want to send just one event, use text/plain as Content Type as usual.
If you want to send many events, use application/xml or application/json as Content Type. Then bundle the events into a xml or json body with an "eventbundle" root with N "event" child elements. See examples in wiki.
